README.md

```diff
@@ -1,68 +1,96 @@
 
-Collection of Python Tools for [Path of Exile](https://www.pathofexile.com/).
 
-More detailed docs: [http://omegak2.net/poe/PyPoE/](http://omegak2.net/poe/PyPoE/)
+## How does this work?
 
-The docs will be updated soon with details on how to update the specification for new leagues.
+Each new Path of Exile league has an updated game data file, which has to be parsed out and mined for information about the items, monsters, league mechanics and changes to the core and secondary/tertiary game mechanics which then need to make their way into the [PoE Wiki](https://www.poewiki.net).
 
+These tools rely on specification files which are able to parse and read the `*.dat` files contained in the main game data file in order to determine what type of data holds what value at any given time and how to translate this into a format the Wiki can understand. These change patch to patch and have to be updated.
 
-Common Problems & Advisory
---------
-* Install **Python 3.7** for maximum compatibility:
-* **UI will be reworked for bundle support and is not functional at the moment**
-* On Windows 10 machines there seems to a be bug in the Python installation that prevents arguments being passed to the command line interface; you can identify this issue if you get a "help" listing if you supplied more then 1 argument. See [this on stack overflow](https://stackoverflow.com/questions/2640971/windows-is-not-passing-command-line-arguments-to-python-programs-executed-from-t) for possible solutions
+More detailed docs: [http://omegak2.net/poe/PyPoE/](http://omegak2.net/poe/PyPoE/)
 
 
-Overview
---------
-Parts:
+## Overview
+
 * Library toolkit for programmers (PyPoE/poe)
-* UI based on Qt for browsing the game files (currently not working)
+* UI based on Qt for browsing the game files (currently not working) -- marked for deprecation
 * CLI interface for extracting/exporting data (for the wiki, more TBD)
 
-Resources
--------
-* Discord: [Project Path of Exile Wiki](https://discord.gg/CE46HADc5T)
+## Getting Started
 
-Important Notes
---------
-Alpha Stage:
-* Code structure and in particular the API may change at any time
-* Incomplete in many areas (check files and TODOs)
-* Tests still have to be written for a lot of things.
-* Many functions and classes are not yet fully documented
+1. Install:
 
-Dev branch:
-* Broken code may be committed occasionally to the dev branch
+    - Python 3.11 - https://www.python.org/downloads/release/python-3110/
```

Comment:
Mentioning `pyenv` as a way to manage Python installations on distros whose baseline is worse could be neat.